iis-wf/iis-wf-affmatching/src/test/java/eu/dnetlib/iis/wf/affmatching/match/voter/CommonAffSectionWordsVoterTest.java

```diff
@@ -0,0 +1,226 @@
+
+import static com.google.common.collect.ImmutableList.of;
+import static com.google.common.collect.Lists.newArrayList;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import java.util.function.Function;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Lists;
+
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchAffiliation;
+import eu.dnetlib.iis.wf.affmatching.model.AffMatchOrganization;
+import eu.dnetlib.iis.wf.affmatching.orgsection.OrganizationSectionsSplitter;
+
+/**
+ * @author madryk, Łukasz Dumiszewski
+ */
+@RunWith(MockitoJUnitRunner.class)
+public class CommonAffSectionWordsVoterTest {
+
+    
+    private double MIN_COMMON_WORDS_RATIO = 0.8;
+    
+    private int WORDS_TO_REMOVE_MAX_LENGTH = 2;
+    
+    private List<Character> CHARS_TO_FILTER = ImmutableList.of(',');
+    
+    
+    @InjectMocks
+    private CommonAffSectionWordsVoter voter = new CommonAffSectionWordsVoter(CHARS_TO_FILTER, WORDS_TO_REMOVE_MAX_LENGTH, MIN_COMMON_WORDS_RATIO);
+    
+    @Mock
+    private OrganizationSectionsSplitter organizationSectionsSplitter;
+    
+    @Mock
+    private StringFilter stringFilter;
+    
+    @Mock
+    private CommonSimilarWordCalculator commonSimilarWordCalculator;
+    
+    @Mock
+    private Function<AffMatchOrganization, List<String>> getOrgNamesFunction;
+    
+    
+    private AffMatchAffiliation affiliation = new AffMatchAffiliation("DOC_ID", 1);
+    
+    private String affOrgName = "AFF_ORG_NAME";
+    
+    private AffMatchOrganization organization = new AffMatchOrganization("ORG_ID");
+    
+    
+    @Before
+    public void setup() {
+        
+        affiliation.setOrganizationName(affOrgName);
+           
+    }
+    
+    
+    //------------------------ TESTS --------------------------
+    
+    @Test(expected = NullPointerException.class)
+    public void constructor_NULL_CHARS_TO_FILTER() {
+        
+        // execute
+        
+        new CommonAffSectionWordsVoter(null, 2, 0.8);
+    
+    }
+    
+    
+    @Test(expected = IllegalArgumentException.class)
+    public void constructor_NEGATIVE_MIN_WORD_LENGTH() {
+
+        // execute
+        
+        new CommonAffSectionWordsVoter(of(), -1, 0.8);
+        
+    }
+    
+    
+    @Test(expected = IllegalArgumentException.class)
+    public void constructor_TOO_HIGH_MIN_COMMON_WORDS_RATIO() {
+        
+        // execute
+        
+        new CommonAffSectionWordsVoter(of(), 2, 1.1);
+        
+    }
+    
+    
+    @Test(expected = IllegalArgumentException.class)
+    public void constructor_TOO_LOW_MIN_COMMON_WORDS_PERCENTAGE() {
```

Comment:
`TOO_LOW_MIN_COMMON_WORDS_PERCENTAGE` should actually be a `TOO_LOW_MIN_COMMON_WORDS_RATIO` to be consistent with naming convention